action/lib/parse.js

```diff
@@ -24,18 +50,77 @@ export default function (title, target) {
     return process.exit(0) // soft exit
   }
 
+  let isDev = devDependencyRegEx.test(title);
+  let isProd = dependencyRegEx.test(title);
+  const isSecurity = securityRegEx.test(title) || labels.includes("security") || labels.includes("Security");
+
+  if (!isDev && !isProd) {
```

Comment:
two points of view here:

- why not read `package.json` as the primary (and actually source of truth if it exists)
- this will not work for non-javascript / npm based projects, whereas the action thus far has been agnostic

Reply:
see comment above